3.3/test/django-test-apps/no-static/run.py

```diff
@@ -0,0 +1,2 @@
+import os
+os.execl('./manage.py', './manage.py', 'runserver', '0.0.0.0:8080')
```

Comment:
IIRC my intention here was to have less dependencies. It's a pain already to run tests because it takes 2~4 min each run.
Probably the time saved for not using `gunicorn` here is negligible, so that it is not a very compelling trade off in that direction.
However, this does add some value by testing the `.sti/environment:APP_FILE` way of running the app. @soltysh what do you think about that?